asteroid/models/base_models.py

```diff
@@ -177,13 +185,15 @@ def get_model_args(self):
         raise NotImplementedError
 
 
-class BaseTasNet(BaseModel):
+class BaseEMDNet(BaseModel):
```

Comment:
@popcornell Opinion about renaming that to `BaseEMDNet`? 
(with code to make it backward compatible, obviously)